ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/isis-interfaces.j2

```diff
@@ -0,0 +1,44 @@
+{# isis interfaces #}
+{% if router_isis is defined and router_isis is not none %}
+### ISIS interfaces Summary
+
+| Interface | ISIS instance | ISIS metric | Interface mode | 
+| -------- | -------- | -------- | -------- | 
+{#- ethernet interfaces -#}
+{%-      if ethernet_interfaces is defined and ethernet_interfaces is not none %}
+{%          for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %} 
+{%-              if ethernet_interfaces[ethernet_interface].isis_enable is defined and ethernet_interfaces[ethernet_interface].isis_enable is not none %} 
+{{  ethernet_interface }} | {{ ethernet_interfaces[ethernet_interface].isis_enable }} |  {{ ethernet_interfaces[ethernet_interface].isis_metric }} |
```

Comment:
First `|` is missing and lead to broken structure for the table.

Reply:
fixed